src/domain/mainwindow.py

```diff
@@ -545,8 +575,10 @@ def update_en_actuator_list(self):
 
         self.ui.listWidget_en_actuators.clear()
         for actuator in Actuator.actuators:
-            if actuator.enabled:
-                text = f"({actuator.type.value}) {actuator.id}"
+            if Actuator.actuators[actuator].enabled:
+                text = (
+                    f"({Actuator.actuators[actuator].type.value}) {Actuator.actuators[actuator].id}"
+                )
```

Comment:
for actuator in Actuator.actuators.values():
            if actuator.enabled:
                text = f"({actuator.type.value}) {actuator.id}"